### PR TITLE
Implementation of the `File Upload` button

### PR DESCRIPTION
**Pull request by Oleh-Stefanovych**
Now files page has a `Upload Files` button.

The `ESLint` result:
image

### CODE REVIEW

**Comment by Oleh-Stefanovych** [comment]
@dkalinin, PTAL.

File: app/component/high-order-component/files-page/index.js [hunk 1]
```diff
@@ -70,14 +71,34 @@ export class FilesPage extends HighOrderComponent {
...
 
     new Button(wrapper.element, {
       text: 'New Folder',
     }).onClick(() => {
       this.store.dispatch('startFolderCreation');
     });
 
+    const inputComponent = new Input(wrapper.element, {
+      type: 'file',
+      isMultiple: true,
```
Comment (dkalinin) [inline app/component/high-order-component/files-page/index.js]: Please add class.

File: app/component/high-order-component/files-page/index.js [hunk 2]
```diff
@@ -70,14 +71,34 @@ export class FilesPage extends HighOrderComponent {
...
+    }).onChange(() => {
+      this.store.dispatch('uploadFiles', inputComponent.files, this.store.state.currentFolder)
+          .then(() => {
+            this.store.dispatch('fetchFiles', this.store.state.currentFolder)
+                .catch((error) => this._showServerMessage(error.message));
```
Comment (dkalinin) [inline app/component/high-order-component/files-page/index.js]: This is not needed.

**Comment by Oleh-Stefanovych** [comment]
@dkalinin, PTAL.

**Pull request merged**